app/views/admin/items/index.html.erb

```diff
@@ -12,9 +12,9 @@
               <!-- <p>Description: <%= item.description %></p> -->
               <footer>
                 <p class='small'>Price: $<%= item.price %></p>
-                <p><%= link_to "Edit", edit_admin_item_path(item), class: "badge badge-info"  %></p>
+                <h5><%= link_to "Edit", edit_admin_item_path(item), class: "badge badge-secondary"  %></h5>
                 <% unless item.retired? %>
```

Comment:
Feel like the edit button here is hard to read, we should also update the retired tag to be sized more like the button

Reply:
I went ahead and updated that badge and lightened the background